README.md

```diff
@@ -0,0 +1,6 @@
+*Курилович Карина 953504*
```

Comment:
<img width="327" alt="Screenshot 2022-04-21 at 21 22 12" src="https://user-images.githubusercontent.com/43992068/164546179-45127fbd-40e7-4c3f-8b0f-b68cce401fcb.png">
It still doesn't have a mobile version.